### PR TITLE
Michael Real React-Router-Movies

### PR DESCRIPTION
**Pull request by Michael-Real**

### CODE REVIEW

**Comment by bdurb** [comment]
@Michael-Real thank you for the PR.

File: client/src/Movies/Movie.js [hunk 1]
```diff
@@ -10,9 +10,7 @@ export default class Movie extends Component {
   }
 
   componentDidMount() {
-    // change this line to grab the id passed on the URL
-    const id = 1;
-    this.fetchMovie(id);
+    this.fetchMovie(this.props.match.params.id)
```
Comment (bdurb) [inline client/src/Movies/Movie.js]: 👍

File: client/src/Movies/MovieCard.js [hunk 2]
```diff
@@ -1,7 +1,25 @@
 import React from 'react';
 
 const MovieCard = props => {
-  return;
+const {title, director, metascore, stars} = props.movie;
```
Comment (bdurb) [inline client/src/Movies/MovieCard.js]: nice work on moviecard refactor

**Comment by bdurb** [comment]
@Michael-Real nice job on this project.

File: client/src/Movies/MovieCard.js (hunk 2 above)
Comment (Michael-Real) [reply client/src/Movies/MovieCard.js]: Just realized it was stretch

**Pull request closed without merging**